.github/workflows/deploy-sfdx-project-to-developer.yml

```
@@ -130,6 +130,22 @@ on:
         description: Instruct Salesforce to apply destructive changes (i.e. deletes) after the deployment. This is the default. In some cases you may want to apply destructive changes before deploying. In that case, set this input to false.
         required: false
         default: true
+      environment:
+        type: string
+        description: The name of the environment in the repository that determines who is authorized to approve deployments to this environment.
+        required: false
+      metadata-zip-file:
+        type: string
+        description: This input is not required, but if set it is required that both `test-classes-file` and `deletes-file` are also set.
+        required: false
+      test-classes-file:
+        type: string
+        description: This input is not required, but if set it is required that both `metadata-zip-file` and `deletes-file` are also set.
+        required: false
+      deletes-file:
+        type: string
+        description: This input is not required, but if set it is required that both `metadata-zip-file` and `test-classes-file` are also set.
```

Add explanation of what the input is for as well.

```suggestion
        description: The name of an artifact that was previously uploaded in this workflow and that contains a single CSV file called `deletes.csv` with information about metadata marked for deletion. If not set, the workflow will build an ad-hoc metadata bundle which will also produce this file. If this input is specified, `metadata-artifact` and `test-classes-artifact` must also be specified.

```